stacker/providers/aws/default.py

```diff
@@ -106,105 +273,71 @@ def cloudformation(self):
 
         return self._cloudformation
 
-    def get_stack(self, stack_name, **kwargs):
-        try:
-            return retry_on_throttling(
-                self.cloudformation.describe_stacks,
-                kwargs=dict(StackName=stack_name))['Stacks'][0]
-        except botocore.exceptions.ClientError as e:
-            if "does not exist" not in e.message:
-                raise
-            raise exceptions.StackDoesNotExist(stack_name)
+    @property
+    def listener(self):
+        # deals w/ multiprocessing issues w/ sharing ssl conns
+        # see https://github.com/remind101/stacker/issues/196
+        pid = os.getpid()
+        if pid != self._pid or not self._listener:
```

Comment:
Hmm - this is going to cause issues when/if this goes multiprocess.  I wonder if we need this code anymore, if you're removing multiprocessing?

Anyway, since it checks if `pid` is not `self._pid` you could end up with multiple listeners, using different SQS queues, if we use multiprocessing.